src/components/test/UserAvatar.test.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { shallow } from "enzyme";
+import UserAvatar from "../UserAvatar";
+import { getPrefabProfile } from "../../test/testProfiles";
+
+describe("UserAvatar", () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+  it("renders without crashing", () => {
+    expect(() => {
+      shallow(<UserAvatar profile={getPrefabProfile(0)} avatarSize="large" />);
+    }).not.toThrow();
+  });
+  it("matches snapshot", () => {
+    const component = shallow(
+      <UserAvatar profile={getPrefabProfile(0)} avatarSize="large" />
+    );
+    expect(component).toMatchSnapshot();
```

Comment:
Would like to see tests other than snapshot whenever possible since they break more easily.

Reply:
@Ajaxan maybe we just remove the snapshot altogether and just leave it as does not throw an error.  Not really sure what else to test.